src/components/AddBareMetalHosts/BareMetalInventoryAddHost.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Text, TextContent, Button } from '@patternfly/react-core';
+import HostsTable from '../hosts/HostsTable';
+import { Cluster, HostRequirements as HostRequirementsType } from '../../api/types';
+import HostRequirements from '../fetching/HostRequirements';
+import { DiscoveryImageModalButton } from '../clusterConfiguration/discoveryImageModal';
+import {
+  DiscoveryTroubleshootingModal,
+  HostsNotShowingLink,
+} from '../clusterConfiguration/DiscoveryTroubleshootingModal';
+
+const HostRequirementsContent = ({ worker = {} }: { worker?: HostRequirementsType['worker'] }) => (
+  <Text component="p">
+    Worker hosts are required to be equiped by at least {worker.cpuCores || 2} CPU cores,{' '}
+    {worker.ramGib || 8} GB of RAM, and {worker.diskSizeGb || 120}
+    GB of filesystem storage each.
+  </Text>
+);
+
+const BaremetalInventoryAddHosts: React.FC<{ cluster: Cluster }> = ({ cluster }) => {
+  const [isDiscoveryHintModalOpen, setDiscoveryHintModalOpen] = React.useState(false);
+
+  return (
+    <>
+      <TextContent>
+        <Text component="h2">Bare Metal Inventory</Text>
+        <Text component="p">
+          <DiscoveryImageModalButton ButtonComponent={Button} cluster={cluster} />
+        </Text>
+        <Text component="p">
```

Comment:
These are adepts for reusable components but contemplating if it is not over-killing.